exopy_hqc_legacy/tasks/tasks/instr/views/run_awg_views.enaml

```diff
@@ -40,5 +40,10 @@ enamldef RunAWGView(InstrView): view:
             hug_width = 'ignore'
             text := task.switch
             entries_updater << task.list_accessible_database_entries
-            tool_tip = EVALUATER_TOOLTIP
+            tool_tip = 'Should be On, Off or Event'
```

Comment:
It would be best to preserve the evaluator tooltip since the user can use a formula. You can do:
```
tool_tip = 'Should be On, Off or Event\n\n' + EVALUATER_TOOLTIP
```